### PR TITLE
doc: how to reply to a message

### LINKED ISSUE #187: Support thread_ts for threaded updates to a single topic
### Description

Describe your issue here.

### What type of issue is this? (place an `x` in one of the `[ ]`)
- [ ] bug
- [ ] enhancement (feature request)
- [ ] question
- [x] documentation related
- [x] example code related
- [ ] testing related
- [ ] discussion

### Requirements (place an `x` in each of the `[ ]`)
* [x] I've read and understood the [Contributing guidelines](https://github.com/slackapi/slack-github-action/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

---

I would like the ability to gather updates from a long build+deploy process into a single thread. I'd love to use this (or a similar) pre-built action to accomplish this, but I don't see any support (input parameter?) for `thread_ts`. I know the Slack API supports that functionality; is it possible to extend this GitHub Action to leverage that feature?

### PR DESCRIPTION
**Pull request by WilliamBergamin**
###  Summary

This resolved #187 

This PR adds documentation describing how to reply to an existing message and a unit test to ensure the behavior is maintained

### Requirements

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/slack-github-action/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: README.md
Comment (filmaj) [inline README.md:222]: Let's give this a more descriptive ID so the association between the step ID and the use of that ID to get its output in a later step is clearer. Maybe:

```suggestion
- id: deployment_message
```

... and then using `"thread_ts": "${{ steps.deployment_message.outputs.ts }}"` later in the example.

**Review by filmaj — APPROVED** [review]
Left a few suggestions for the README but other than that LGTM!

**Review by zimeg — APPROVED** [review]
🙌 This is great! Found something small in the payload that might need changing before a merge, but the instructions read well and I find this super helpful - I was excited to find a threaded message! 🚀

**Comment by zimeg** [comment]
🤔 I'm also open to updating the example, but no blocker at all. The deployment "started/completed" makes sense for updating a message (it makes sense for threading too), but wondering if this could post "a new commit was made" with commit details in thread instead? Or extending the current example to also `reply_broadcast` the message to channel?

Just some thoughts, feel free to ignore!

**Comment by fleytman** [comment]
Can u publicate new release of slack-github-action with these changes?

**Comment by zimeg** [comment]
@fleytman @WilliamBergamin called it with this being available in the current version `v1.26.0` (and a few releases before if I had to guess). This [example in the README](https://github.com/slackapi/slack-github-action#reply-to-a-message) should be good to paste into a workflow you have, taking care to note the `thread_ts` value!

No other fixes or enhancements have landed since this current release and this appears in the current README, so I think we're alright to wait for another release for now 🚀

**Pull request merged**
